### PR TITLE
Adds pipeline configuration

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: configuration/coding_plans.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    else:
+        return Codes.NOT_CODED
+
+
+def get_rqa_coding_plans(pipeline_name):
+    return [
+        CodingPlan(raw_field="rqa_s01e01_raw",
+                   time_field="sent_on",
+                   run_id_field="rqa_s01e01_run_id",
+                   coda_filename="Test_Daniel_pipeline_s01e01.json",
```
Comment (IsaackMwenda) [inline configuration/coding_plans.py]:
```suggestion
                   coda_filename="Test-Pipeline-Daniel_s01e01.json",
```

File: configuration/pipeline_config.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+  "PipelineName": "Test-Pipeline-Daniel",
+  "RawDataSources": [
+    {
+      "SourceType": "RapidPro",
+      "Domain": "textit.in",
+      "TokenFileURL": "gs://avf-credentials/experimental-test-text-it-token.txt",
```
Comment (IsaackMwenda) [inline configuration/pipeline_config.json]:
```suggestion
      "TokenFileURL": "gs://avf-credentials/experimental-test-textit-token.txt",
```

Comment (IsaackMwenda) [reply configuration/pipeline_config.json]: rename gs file too for consistency with new pipeline names

File: configuration/coding_plans.py (hunk 1 above)
Comment (IsaackMwenda) [reply configuration/coding_plans.py]: and elsewhere

**Review by IsaackMwenda — APPROVED** [review]

File: configuration/coding_plans.py (hunk 1 above)
Comment (DanielMwendwa) [reply configuration/coding_plans.py]: thanks, updated

File: configuration/pipeline_config.json (hunk 2 above)
Comment (DanielMwendwa) [reply configuration/pipeline_config.json]: updated thanks

**Pull request merged**